packages/react-components/react-radio/src/components/Radio/useRadioStyles.ts

```diff
@@ -116,6 +115,15 @@ const useInputStyles = makeStyles({
       },
     },
   },
+
+  after: {
+    height: '100%',
+    width: `calc(${indicatorSize} + 2 * ${tokens.spacingHorizontalS})`,
+  },
+  below: {
+    height: `calc(${indicatorSize} + ${tokens.spacingVerticalS})`,
```

Comment:
Does this need to account for top and bottom padding as well?
```suggestion
    height: `calc(${indicatorSize} + 2 * ${tokens.spacingVerticalS})`,
```

Reply:
Good catch, changed.